_includes/tutorials/column-difference/ksql/code/tutorial-steps/dev/query-with-column-difference.sql

```diff
@@ -0,0 +1,6 @@
+SELECT FIRST_NAME,
+       LAST_NAME,
+       CURRENT_PURCHASE - PREVIOUS_PURCHASE as PURCHASE_DIFF
+       
```

Comment:
Could you please remove some of these white lines? It looks odd in the rendering for the "Short Answer" section.